sql/sql_yacc.yy

```diff
@@ -444,7 +444,7 @@ void warn_about_deprecated_national(THD *thd)
   1. We do not accept any reduce/reduce conflicts
   2. We should not introduce new shift/reduce conflicts any more.
 */
-%expect 107
+%expect 109
```

Comment:
the increase 2 indicates?

Reply:
I've added into_clause to two new places in query_expression. As the commend above select_stmt_with_into explains, this is ambiguous.